app/main.py

```diff
@@ -1,8 +1,24 @@
 class Person:
-    # write your code here
-    pass
+
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    [Person(person["name"], person["age"]) for person in people]
```

Comment:
The list comprehension creates `Person` instances but does not store or return them. This line effectively does nothing because the list is not assigned to a variable or returned.